src/commands/smembers.json

```
@@ -35,6 +35,13 @@
                 }
             }
         ],
+        "reply_schema": {
+            "type": "array",
+            "description": "All elements of the set.",
+            "items": {
```

```suggestion
            "uniqueItems": true,
            "items": {
```

@guybe7 @fadidahanna 

https://github.com/redis/redis/blob/5b61b0dc6d2579ee484fa6cf29bfac59513f84ab/tests/integration/corrupt-dump.tcl#L827

Normally, we can tag this command with `uniqueItems` but we have the above test which reads a set from the rdb with duplicate elements. So, if we add `uniqueItems`, it fails the json validation. Leaving decision up to you.

---

k, in that case suggest keeping "items". thanks for pointing out the issue.